themes/codeMirror/customThemes/darkbrewery-v301.css

```diff
@@ -0,0 +1,122 @@
+/* Main BG color and normal text color */
+.CodeMirror {
+    background: #293134;
+    color: #91A6AA;
+}
+
+/* Brew BG */
+.brewRenderer {
+    background-color: #293134;
+}
+/* Blinking cursor */
+.CodeMirror-cursor {
+    border-left: 1px solid #e0e2e4;
+}
+/* Bars at the top */
+.snippetBar {
+    background-color: #2F393C;
+    color: white;
+}
+nav {
+    background-color: #293134;
+}
+nav .navItem {
+    background-color: #293134;
+}
+/* Fix for Homebrewery custom Snippet icons */
+.snippetBar .fac {
+    filter: invert(1);
+}
+.snippetBar .snippetGroup .dropdown {
+    background-color: #2F393C;
+}
+/* Line number stuff */
+    .CodeMirror-gutter-elt {
+    color: #81969A;
+}
+.CodeMirror-linenumber {
+    background-color: #293134;
+}
+.CodeMirror-gutter {
+    background-color: #293134;
+}
+/* column splits */
+.editor .codeEditor .columnSplit {
+    font-style: italic;
+    color: inherit;
+    background-color:#1f5763;
+    border-bottom: #299 solid 1px;
+}
+/* Colors for headings and such */
+.cm-s-default .cm-header {
+    color: #c51b1b;
+    -webkit-text-stroke-width: 0.1px; 
+    -webkit-text-stroke-color: #000; 
+    /* ###Headings */
```

Comment:
There are several places in this CSS where the comment is coming *after* the content it describes, though it is still inside the relevant brackets.  This really threw me off.  I think all those comments should come before their relevant context.

Reply:
Moved the comments to be consistently placed.